tmtools/helpers.py

```diff
@@ -0,0 +1,77 @@
+"""Helper tools to work with TM-Align."""
+
+from Bio.PDB.Structure import Structure
+import numpy as np
+
+
+def apply_tmalign_transformation(original_structure, tmalign_result):
+    """
+    Transform structure based on TM-Align result.
+
+    This function creates a copy of the input structure and applies the rotation
+    matrix and translation vector from a TM-align result to all atoms in the copied
+    structure.
+
+    Parameters
+    ----------
+    original_structure : Bio.PDB.Structure.Structure
```

Comment:
Question: what is more appropriate, representing the structure as a NumPy array, or as a BioPython `Structure` instance? The former has the advantage that it avoids pulling in BioPython, but maybe the latter is more natural.

Reply:
I would provide both, but the second is more natural for the intended use of TMalign (since the aligned structures are what is "interesting"). I will implement both